Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Utilities.ps1

```diff
@@ -894,3 +877,109 @@ function Invoke-ActionWithDefaultRetries
 
     Invoke-ActionWithRetries @parameters
 }
+
+function Test-OldSdk
+{
+    $sdkVersionString = (Get-SfSdkVersion)
+    if ($sdkVersionString -match "^\d+\.\d+")
```

Comment:
does it work if sdkVersionString is null

Reply:
yes. it return false, does not throw